### ISSUE TITLE
Large Links, Buttons, and Controls: What needs to happen for this to work?

### ISSUE THREAD

**Issue by haltersweb** [w3c/wai-showcase-examples#200]
I don't find this is usually the issue so I suggest we delete this line: "In many cases simply re-arranging and better positioning the controls improves ease of clicking and tapping." (Plus I think this line takes away from the more important issue of large click targets)

I would reword the following:

"Enlarging the size of the control or activation areas around the controls brings improvement (just increasing the font size of the control often isn't enough). Additionally, providing labels for controls, which is an accessibility requirement itself, often enlarges the activation area, as does nesting the controls within their corresponding label."

**Comment by nitedog** [comment]
Updated as suggested, except for "as does nesting the controls within their corresponding label". This is way too technical and jargony IMO. It assumed knowledge of HTML markup and nesting elements.
